lib/openshift/subscription.rb

```diff
@@ -12,6 +12,11 @@ def startingCSV(user: nil, quiet: false, cached: false)
       return rr.dig('status', 'currentCSV')      
     end
 
+    def installedCSV(user: nil, quiet: false, cached: false)
```

Comment:
Perhaps make quiet/cached defaults to true ?

Reply:
@liangxia good point, I agree `quiet` should be `true` by default, however I'm not sure if `cached` should be true or false by default, anyway let me update all to `true` by default and see if any issues